src/App.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import TodoPage from './TodoPage';
+
+function App() {
+  const [state, setState] = useState({
+    todo: '',
+    todos: [],
+    isEmpty: true,
+  });
+
+  const { todo, todos, isEmpty } = state;
+
+  function handleClickAdd() {
+    if (todo === '') {
+      return;
+    }
+
+    todos.push(todo);
+
+    setState({
+      ...state,
+      todo: '',
+      isEmpty: false,
+    });
+  }
+
+  function handleClickFinish(index) {
+    todos.splice(index, 1);
+
+    const isEmptyCheck = (todos.length === 0);
+
+    setState({
+      ...state,
+      isEmpty: isEmptyCheck,
```

Comment:
```suggestion
      isEmpty: todos.length === 0,
```
이 경우엔 인라인으로 처리해도 될 것 같네요~